### PR TITLE
Ecto 2.1

### PR DESCRIPTION
**Pull request by mbenatti**
Support for Ecto 2.1

### CODE REVIEW

**Comment by ankhers** [comment]
Thanks for this! I should be able to take a look at it later today.

File: lib/mongo_ecto.ex [hunk 1]
```diff
@@ -537,13 +541,13 @@ defmodule Mongo.Ecto do
...
   @doc false
-  def insert(_repo, meta, _params, [_|_] = returning, _opts) do
+  def insert(_repo, meta, _params, _on_conflict, [_|_] = returning, _opts) do
     raise ArgumentError,
       "MongoDB adapter does not support :read_after_writes in models. " <>
       "The following fields in #{inspect meta.schema} are tagged as such: #{inspect returning}"
   end
 
-  def insert(repo, meta, params, [], opts) do
+  def insert(repo, meta, params, _, [], opts) do
```
Comment (ankhers) [inline lib/mongo_ecto.ex]: You appear to be ignoring the `on_conflict` param. Is there something we should be doing with it?

**Comment by ankhers** [comment]
Can you also try updating all of the dependencies to the latest version (specifically the `mongodb` package for 0.3)?

File: lib/mongo_ecto.ex (hunk 1 above)
Comment (mbenatti) [inline lib/mongo_ecto.ex]: Its just ignore, I don't know the behavior of mongodb in that case, we can discuss that...

**Comment by ankhers** [comment]
The tests are also failing because Ecto 2.1 dropped support for Elixir 1.2. Would you be able to update the travis file and `mix.exs` to show that?

**Comment by ankhers** [comment]
Thanks for the start here. I am closing this because I manually fixed the conflicts and will soon push the changes to the `ecto-2.1` branch.

**Comment by mbenatti** [comment]
Thanks @ankhers 
I'm short on time this week to get attention to this project

**Pull request closed without merging**